libs/libarchfpga/src/read_fpga_interchange_arch.cpp

```diff
@@ -1533,7 +1533,7 @@ struct ArchReader {
         //        the RR graph generation is correct.
         //        This can be removed once the RR graph reader from the interchange
         //        device is ready and functional.
-        int num_seg = 1; //wire_names.size();
+        int num_seg = wire_names.size();
```

Comment:
I believe that also the `if (index >= num_seg) break;` line below should be removed, right?